extensions/amp-consent/0.1/amp-consent.js

```diff
@@ -517,6 +541,49 @@ export class AmpConsent extends AMP.BaseElement {
     });
   }
 
+  /**
+   * Sync consent string and consent state value to local storage.
+   */
+  syncConsentStringAndStateValue_() {
+    const responsePromise = this.getConsentRemote_();
+    responsePromise.then(response => {
+      if (
+        response &&
+        typeof this.consentConfig_['consentRequired'] === 'boolean' &&
```

Comment:
why do we need to check this?

Reply:
This is unnecessary now that we'll only call this once.